schema/aggregations/aggregation_count.js

```diff
@@ -19,7 +22,7 @@ export const AggregationCountType = new GraphQLObjectType({
   },
 });
 
-export default {
+export default ({
   type: AggregationCountType,
   resolve: ({ name, count }, id) => ({ id, name, count }),
-};
+}: GraphQLFieldConfig<AggregationCountType, any>);
```

Comment:
ah yeah, I wanted all to be `*` based on the default, but I really can't seem to find any documentation on `*` vs `any`.